src/rust/engine/src/scheduler.rs

```diff
@@ -88,15 +105,20 @@ impl Session {
     should_report_workunits: bool,
     session_values: Value,
   ) -> Session {
-    let workunit_store = WorkunitStore::new(should_render_ui);
-    let display = if should_render_ui {
-      Some(Mutex::new(ConsoleUI::new(
+    let workunit_store = WorkunitStore::new(!should_render_ui);
+    let display = Mutex::new(if should_render_ui {
+      SessionDisplay::ConsoleUI(ConsoleUI::new(
         workunit_store.clone(),
         scheduler.core.local_parallelism,
-      )))
+      ))
     } else {
-      None
-    };
+      SessionDisplay::Logging {
+        // TODO: This threshold should likely be configurable, but the interval we render at
+        // probably does not need to be.
+        straggler_threshold: Duration::from_secs(60),
```

Comment:
To confirm, this will only log if the workunit has been taking > 60 seconds? I wonder if we want this to be lower, maybe 45 seconds.

Reply:
I expect that in a CI environment, 60 is sufficient. Let's try it out for a while and see whether it is responsive enough.